components/raftstore-v2/src/fsm/store.rs

```diff
@@ -84,9 +123,29 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T> StoreFsmDelegate<'a, EK, ER, T> {
         Self { fsm, store_ctx }
     }
 
-    pub fn handle_msgs(&self, store_msg_buf: &mut Vec<StoreMsg>) {
+    fn on_start(&mut self) {
+        if self.fsm.store.start_time.is_some() {
+            panic!("{:?} unable to start again", self.fsm.store.logger.list(),);
+        }
+
+        self.fsm.store.start_time = Some(
+            SystemTime::now()
```

Comment:
Why not use `tikv_util::Instant`?

Reply:
Because we need unix timestamp. And it will be encoded to PD in the end.